pypesto/ensemble/ensemble.py

```diff
@@ -616,14 +618,26 @@ def from_optimization_endpoints(
             A pyPESTO result that contains an optimization result.
         cutoff:
             Exclude parameters from the optimization if the
-            nllh is higher than the `cutoff`.
+            nllh is higher than the `cutoff`. If cutoff is None, a cutoff
+            will be calculated based on the percentile passed.
```

Comment:
Describe that `cutoff` is actually an absolute threshold on the likelihood. e.g. to get all vectors s.t. `fval(vector) <= fval(optimal_vector) + 5`, specify `cutoff=fval(optimal_vector) + 5`.

Reply:
Would prefer if `cutoff` could be a relative difference in `fval`. e.g.
`cutoff=5`
instead of
`cutoff=fval(optimal_vector) + 5`

This perhaps makes more sense since a user may then calculate the cutoff independently and in advance of the optimization result, e.g. the following is independent of the optimization result.
```python
    df = result.problem.dim
    range = chi2.ppf(q=percentile, df=df)
    cutoff = range
```